Installation does not support Windows (Electron is not compatible to Windows)

While trying to install Brave on Windows, I faced the problem that apparently electron installer does not support Windows. The full error report is below:

npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: electron-installer-redhat@0.2.0
npm ERR! notsup Valid OS:    darwin,linux
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   win32
npm ERR! notsup Actual Arch: x64

--- a/package.json
+++ b/package.json
@@ -94,8 +94,6 @@
     "co-mocha": "^1.1.2",
     "css-loader": "^0.23.0",
     "electron-builder": "^2.3.1",
-    "electron-installer-debian": "^0.2.0",
-    "electron-installer-redhat": "^0.2.0",
     "electron-installer-squirrel-windows": "^1.2.2",
     "electron-packager": "brave/electron-packager",
     "electron-rebuild": "^1.1.1",
@@ -121,6 +119,10 @@
     "webpack-dev-server": "^1.14.0",
     "webpack-notifier": "^1.2.1",
     "xml2js": "^0.4.15"
+  },
+  "optionalDependencies": {
+    "electron-installer-debian": "^0.2.0",
+    "electron-installer-redhat": "^0.2.0"
   },
   "standard": {
     "ignore": [

